core/templates/pages/exploration-editor-page/statistics-tab/templates/state-stats-modal.template.html

```diff
@@ -53,3 +53,11 @@ <h3>Statistics for "<a ng-click="navigateToStateEditor()"><[stateName]></a>"</h3
 <div class="modal-footer">
   <button class="btn btn-secondary" ng-click="cancel()">Close</button>
 </div>
+<style>
+  .modal-body .oppia-quit-card-text {
```

Comment:
The prefix used here (.modal-body) is too generic, please fix.

Reply:
Done.